deepchem/data/datasets.py

```diff
@@ -491,7 +499,7 @@ def gen_data():
 
     return tf.data.Dataset.from_generator(gen_data, dtypes, shapes)
 
-  def make_pytorch_dataset(self, epochs=1, deterministic=False):
+  def make_pytorch_dataset(self, epochs: int = 1, deterministic: bool = False):
```

Comment:
Should we add a return type here?

Reply:
We can't, because the return type is a PyTorch class and we don't want to `import torch` outside the function.